src/software/ai/hl/stp/tactic/tactic_visitor.h

```diff
@@ -46,31 +39,22 @@ class TacticVisitor
      *
      * @param tactic The tactic to visit
      */
-
-    virtual void visit(const ChipTactic &tactic)                 = 0;
+    virtual void visit(const Tactic &tactic)                     = 0;
```

Comment:
we don't want to implement this to avoid accidentally defining a default for tactics and not having a compile-time check

Reply:
removed